handout/components/creating_components.md

```diff
@@ -1,9 +1,11 @@
 # Creating Components
 
-Components in Angular 2 build upon this idea. We define a component's application logic inside a class. To this we then attach a selector and a template.
+Components in Angular 2 build upon the lessons learned from Angular 1.5. We define a component's application logic inside a class. To this we attach metadata using a TypeScript **Decorator**, which is simply a function call that allows the specification of the **selector** and **template**.
 
-- _selector_ is the element property that we use to tell Angular to create and insert an instance of this component.
-- _template_ is a form of HTML that tells Angular how to render this component.
+- _Selector_ is the element property that we use to tell Angular to create and insert an instance of this component.
```

Comment:
`selector` and `template` are property names and should not be capitalized (to reduce confusion)